### PR TITLE
[LYFT] [STRMCMP-1161] Adding way to capture application env to kinesis and s…

### PR DESCRIPTION
**Pull request by premsantosh**
…3 source python sdk

### CODE REVIEW

File: sdks/python/apache_beam/io/lyft/s3_and_kinesis.py [hunk 1]
```diff
@@ -84,6 +85,16 @@ def get_windowing(self, inputs):
     def infer_output_type(self, unused_input_type):
         return bytes
 
+    def with_app_env(self, app_env):
```
Comment (maghamravi) [inline sdks/python/apache_beam/io/lyft/s3_and_kinesis.py]: minor nit: given this is a public api, let's call this `with_app_environment` and avoid short names ?

**Comment by maghamravi** [comment]
Also, can you please extend `s3.py` to take in environment as an argument.

**Comment by premsantosh** [comment]
> Also, can you please extend `s3.py` to take in environment as an argument.

Woops, missed that. Done and rebased.

**Pull request merged**